README.md

```diff
@@ -169,7 +169,7 @@ _Source:_ [What is Docker](https://www.docker.com/why-docker)
 - [Zodiac](https://github.com/CenturyLinkLabs/zodiac) :skull: - A lightweight tool for easy deployment and rollback of dockerized applications. By [@CenturyLinkLabs][centurylinklabs]
 
 ### Deployment and Infrastructure
-
+- [Pull Dog](https://github.com/apps/pull-dog) - A GitHub app that automatically creates Docker-based test environments for your pull requests, from your docker-compose files.
```

Comment:
```suggestion

- [Pull Dog](https://github.com/apps/pull-dog) - A GitHub app that automatically creates Docker-based test environments for your pull requests, from your docker-compose files.
```

Reply:
Thank you, I fixed this. Sorry for missing it the first time!

Reply:
no probemo :)